swampmachine-multimedia/src/main/java/net/kiberion/audio/ChannelControlEvent.java

```diff
@@ -0,0 +1,24 @@
+package net.kiberion.audio;
+
+import org.springframework.context.ApplicationEvent;
+
+import lombok.Getter;
+
+@SuppressWarnings("serial")
+public class ChannelControlEvent extends ApplicationEvent {
```

Comment:
Are you sure you need this class? Can you explain me what is your vision for channel control? Who and when is going to be doing it?